pkg/core/mempool/mem_pool_test.go

```diff
@@ -528,26 +540,88 @@ func TestMempoolAddRemoveConflicts(t *testing.T) {
 	assert.Equal(t, []util.Uint256{tx3.Hash(), tx2.Hash()}, mp.conflicts[tx1.Hash()])
 
 	// tx13 conflicts with tx2, but is not signed by tx2.Sender
-	tx13 := transaction.New([]byte{byte(opcode.PUSH1)}, 0)
-	tx13.NetworkFee = smallNetFee
-	tx13.Nonce = uint32(random.Int(0, 1e4))
-	tx13.Signers = []transaction.Signer{{Account: util.Uint160{3, 2, 1}}}
-	tx13.Attributes = []transaction.Attribute{{
-		Type: transaction.ConflictsT,
-		Value: &transaction.Conflicts{
-			Hash: tx2.Hash(),
-		},
-	}}
+	tx13 := getMalisiousTx(smallNetFee, tx2.Hash())
 	_, ok := mp.TryGetValue(tx13.Hash())
 	require.Equal(t, false, ok)
 	require.ErrorIs(t, mp.Add(tx13, fs), ErrConflictsAttribute)
 
+	// tx15 conflicts with tx14, but added firstly and has the same network fee => tx14 must not be added.
 	tx14 := getConflictsTx(smallNetFee)
 	tx15 := getConflictsTx(smallNetFee, tx14.Hash())
 	require.NoError(t, mp.Add(tx15, fs))
-	require.NoError(t, mp.Add(tx14, fs))
-	err := mp.Add(tx15, fs)
-	require.ErrorIs(t, err, ErrConflictsAttribute)
+	err := mp.Add(tx14, fs)
+	require.Error(t, err)
+	require.True(t, strings.Contains(err.Error(), fmt.Sprintf("conflicting transactions have bigger or equal network fee: %d vs %d", smallNetFee, smallNetFee)))
```

Comment:
Welcome to the `errors.Join()` lovers club.

Reply:
Looking for bumping minimum required version up to go 1.20.